doc/how_to_build_win.md

```diff
@@ -1,20 +1,20 @@
 # Building on Windows
 
-This software can be built using Visual Studio 2015 and Qt 5.9
+This software can be built using Visual Studio 2015 or above and Qt 5.9(later versions still not working correctly.)
```

Comment:
```suggestion
This software can be built using Visual Studio 2015 or above and Qt 5.9 (later Qt versions still not working correctly.)
```
This would fix what I think is a typo (by adding the space) but ensure it still makes sense (the added "Qt") :)

Reply:
Thanks for the feedback and help.